variables.tf

```
@@ -1 +1,90 @@
-# Variables placeholder
+variable "protect_s3_bucket_resources" {
+  description = "S3 bucket resource ARNs to protect from bucket and object deletion"
+  type        = list(string)
+  default     = [""]
+}
+
+variable "protect_iam_role_resources" {
+  description = "IAM role resource ARNs to protect from modification and deletion"
+  type        = list(string)
+  default     = []
+}
+
+variable "allowed_regions" {
+  description = "AWS Regions allowed for use (for use with the restrict regions SCP)"
+  type        = list(string)
+  default     = [""]
+}
+
+variable "target" {
+  description = "id of OU or account on which to attach SCP"
+  type        = string
+}
```

I'm going to suggest you change this to an `object` type variable and have it take the actual OU resource, not the ID of the OU. This will let you get the name of the OU for naming the SCP; so you'd change this to:
```suggestion
variable "target" {
  description = "OU resource to attach SCP"
  type        = object({
    name = string
    id = string
  })
}
```